sleap_roots/scanline.py

```diff
@@ -0,0 +1,62 @@
+"""Get intersections between roots and horizontal scan lines."""
+
+import numpy as np
+import math
+from shapely import LineString
+
+
+def get_pt_scanline(pts: np.ndarray, depth=1080, width=2048, n_line=50) -> list:
```

Comment:
A more descriptive name:
```suggestion
def get_scanline_intersections(pts: np.ndarray, depth: int = 1080, width: int = 2048, n_line: int = 50) -> list:
```